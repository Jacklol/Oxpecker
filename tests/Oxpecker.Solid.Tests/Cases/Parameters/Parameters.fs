module Oxpecker.Solid.Tests.Cases.Arguments

open Oxpecker.Solid

[<SolidComponent>]
let Parameters (id: int) =
    let hello = "Hello "
    div(id=string id, class'="testclass") {
        __() { hello }; "world!"
    }
