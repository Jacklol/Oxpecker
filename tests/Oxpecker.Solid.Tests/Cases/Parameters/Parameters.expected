import { int32ToString } from "./fable_modules/fable-library-js.4.21.0/Util.js";

export function Parameters(id) {
    return <div id={int32ToString(id)}
        class="testclass">
        <>
            Hello 
        </>
        world!
    </div>;
}

